eventmesh-trace-plugin/build.gradle

```diff
@@ -13,5 +13,4 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- */
-spotlessJava.enabled false
\ No newline at end of file
+ */
```

Comment:
Why this config is removed?

Reply:
Because before in order to solve the problem of dependencies, a relatively rude way was used to directly remove several modules from spotless tasks, but now this problem is solved in a general way, and there is no need to remove spotless tasks.